scripts/brk2tmuxp/brk2tmuxp.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+import argparse
```

Comment:
You're an argparser.  I'm more click myself for stuff like this.

Reply:
no real preference, if standard lib do for a little script, I'd rather keep the dependencies to a minimum :)